### PR TITLE
Cadent Aperture MX: alias cadent_aperture_mx to emx_digital

### PR DESCRIPTION
**Pull request by EMXDigital**

### CODE REVIEW

File: static/bidder-info/emx_digital.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 endpoint: "https://hb.emxdgt.com"
 maintainer:
-  email: "adops@emxdigital.com"
+  email: "contactaperturemx@cadent.tv"
```
Comment (gargcreation1992) [inline static/bidder-info/emx_digital.yaml:3]: Sent an email to this email address for verification. Please reply "received".

Comment (gargcreation1992) [reply static/bidder-info/emx_digital.yaml:3]: Is the contact email address for emx_digital to be changed as well?

File: static/bidder-info/cadent_aperture_mx.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    mediaTypes:
+      - banner
+      - video
+  app:
+    mediaTypes:
+      - banner
+      - video
+userSync:
+  iframe:
+    url: "https://cs.emxdgt.com/um?ssp=pbs&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}"
```
Comment (gargcreation1992) [inline static/bidder-info/cadent_aperture_mx.yaml:16]: We tried validating this user sync URL but it does not seem to be working. Could you please check? Here is the URL that we are trying - 
```
https://cs.emxdgt.com/um?ssp=pbs&gdpr=&gdpr_consent=&us_privacy=&redirect=https%3A%2F%2Fib.adnxs.com%2F%2Fsetuid%3Fbidder%3Dcadent_aperture_mx%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%24UID
```

**Review by gargcreation1992 — COMMENTED** [review]
requesting few verifications.

File: static/bidder-info/emx_digital.yaml (hunk 1 above)
Comment (EMXDigital) [reply static/bidder-info/emx_digital.yaml:3]: We've received and replied to the email address.  Both addresses (`cadent_aperture_mx` and `emx_digital`) should be changed to this new @cadent.tv address.

File: static/bidder-info/cadent_aperture_mx.yaml (hunk 2 above)
Comment (gargcreation1992) [reply static/bidder-info/cadent_aperture_mx.yaml:16]: @EMXDigital Requesting to please address this comment.

Comment (EMXDigital) [reply static/bidder-info/cadent_aperture_mx.yaml:16]: @gargcreation1992 we are looking into this now, will post an update shortly.

Comment (EMXDigital) [reply static/bidder-info/cadent_aperture_mx.yaml:16]: @gargcreation1992 We've set the endpoint to return a 204 for the time being.  Please let us know if this is sufficient for now to get this code merged in.

Comment (gargcreation1992) [reply static/bidder-info/cadent_aperture_mx.yaml:16]: @EMXDigital We would like to validate the user sync URL before this PR is merged.

**Comment by gargcreation1992** [comment]
@EMXDigital Requesting you to please address the open comment.

File: static/bidder-info/cadent_aperture_mx.yaml (hunk 2 above)
Comment (EMXDigital) [reply static/bidder-info/cadent_aperture_mx.yaml:16]: @gargcreation1992 we have turned the endpoint back on, so kindly notify if any additional issues arise.  Thanks for the patience.

Comment (gargcreation1992) [reply static/bidder-info/cadent_aperture_mx.yaml:16]: @EMXDigital I am still not able to see UID being set after calling the below endpoint - 
```
https://cs.emxdgt.com/um?ssp=pbs&gdpr=&gdpr_consent=&us_privacy=&redirect=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dcadent_aperture_mx%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%24UID
```

Comment (EMXDigital) [reply static/bidder-info/cadent_aperture_mx.yaml:16]: @gargcreation1992 When I just hit that same endpoint in a browser, I was 302-redirected to:
`https://localhost:8000/setuid?bidder=cadent_aperture_mx&gdpr=&gdpr_consent=&gpp=&gpp_sid=&f=b&uid=[valid value]`

Looks like it's filling in the UID [valid value] with an actual real value on my end.  What are you seeing happening when you call that endpoint?

Comment (gargcreation1992) [reply static/bidder-info/cadent_aperture_mx.yaml:16]: Screenshot 2023-08-01 at 12 07 09 PM

I am getting 204. Here is the screenshot for your reference and below is the URL that I am trying to hit - 
```
https://cs.emxdgt.com/um?ssp=pbs&gdpr=&gdpr_consent=&us_privacy=&redirect=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dcadent_aperture_mx%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%24UID
```

**Comment by EMXDigital** [comment]
Resolved merge conflicts.

File: static/bidder-info/cadent_aperture_mx.yaml (hunk 2 above)
Comment (Sonali-More-Xandr) [reply static/bidder-info/cadent_aperture_mx.yaml:16]: Thank you for your patience on this one @EMXDigital! We have tested this on our end and the url is working fine with North American location.

**Review by gargcreation1992 — APPROVED** [review]

**Pull request merged**